src/cdpTargetsProvider.ts

```diff
@@ -69,5 +86,82 @@ export default class CDPTargetsProvider implements vscode.TreeDataProvider<CDPTa
     public refresh(): void {
         this.telemetryReporter.sendTelemetryEvent("view/refresh");
         this.changeDataEvent.fire(null);
+        this.clearFaviconResourceDirectory();
+    }
+
+    private clearFaviconResourceDirectory() {
+      const fs = require('fs');
+
+      const directory = path.join(this.extensionPath, "resources", "favicons");
+
+      fs.readdir(directory, (readdirError: Error, files: File[]) => {
+        if (readdirError) throw readdirError;
+
+        for (const file of files) {
+          const fileString = file.toString();
+          if (fileString !== ".gitkeep") {
+            fs.unlink(path.join(directory, fileString), (unlinkError: Error) => {
+              if (unlinkError) throw unlinkError;
+            });
+          }
+        }

```

Comment:
can the extension write outside of the folder? Can we add some tests to this scenario?

Reply:
Done!